keps/sig-storage/3141-prevent-volume-mode-conversion/README.md

```diff
@@ -582,11 +636,12 @@ and creating new ones, as well as about cluster-level services (e.g. DNS):
       - Impact of its degraded performance or high-error rates on the feature:
 -->
 
+No.
```

Comment:
this is implemented in the kube-apiserver or in the external-provisioner to emit the event?

Reply:
> It is implemented in the external-provisioner. When any provisioner operations fail, an event is emitted. There's no new dependency added here, but is it right to call that out here anyway?

it's worth explaining how the reporting you suggest users can use can fail.

Reply:
Done